oap-server/server-bootstrap/src/main/resources/oal/browser.oal

```diff
@@ -0,0 +1,73 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+// browser app
```

Comment:
Do we really need so many metrics? Please recheck @arugal @Fine0830. From the UI prototype, it doesn't need so much. Please analysis the metrics used by UI.

Reply:
@wu-sheng @Fine0830  Some percentile associated metrics have been reduced, please review.